wai-extra/Network/Wai/Handler/CGI.hs

```diff
@@ -30,12 +30,7 @@ import qualified Data.CaseInsensitive as CI
 import Data.Monoid (mconcat, mempty, mappend)
 #endif
 
-#if MIN_VERSION_streaming_commons(0, 2, 0)
 import qualified Data.Streaming.ByteString.Builder as Blaze
```

Comment:
A minor issue. But how about changing the `Blaze` prefix?

Reply:
coming right up